stable/mongodb-replicaset/README.md

```diff
@@ -55,22 +55,27 @@ The following table lists the configurable parameters of the mongodb chart and t
-| `metrics.socketTimeout`              | Time to wait for a non-responding socket                                  | `3s`                                                |
-| `metrics.syncTimeout`                | Time an operation with this session will wait before returning an error   | `1m`                                                |
-| `metrics.prometheusServiceDiscovery` | Adds annotations for Prometheus ServiceDiscovery                          | `true`                                              |
+| `metrics.image.repository`          | Image name for metrics exporter                                           | `ssalaues/mongodb-exporter`                         |
+| `metrics.image.tag`                 | Image tag for metrics exporter                                            | `0.6.1`                                             |
+| `metrics.image.pullPolicy`          | Image pull policy for metrics exporter                                    | `IfNotPresent`                                      |
+| `metrics.port`                      | Port for metrics exporter                                                 | `9216`                                              |
+| `metrics.path`                      | URL Path to expose metics                                                 | `/metrics`                                          |
+| `metrics.socketTimeout`             | Time to wait for a non-responding socket                                  | `3s`                                                |
+| `metrics.syncTimeout`               | Time an operation with this session will wait before returning an error   | `1m`                                                |
+| `metrics.prometheusServiceDiscovery`| Adds annotations for Prometheus ServiceDiscovery                          | `true`                                              |
 | `auth.enabled`                      | If `true`, keyfile access control is enabled                              | `false`                                             |
 | `auth.key`                          | Key for internal authentication                                           | ``                                                  |
 | `auth.existingKeySecret`            | If set, an existing secret with this name for the key is used             | ``                                                  |
 | `auth.adminUser`                    | MongoDB admin user                                                        | ``                                                  |
 | `auth.adminPassword`                | MongoDB admin password                                                    | ``                                                  |
+| `auth.dbUser`                       | MongoDB db user                                                           | ``                                                  |
```

Comment:
I feel like it might make sense for this to be grouped under the `db.` namespace rather than `auth.` since its specific to the `db.enabled` being set.

Reply:
Ok, I can change the location.